### PR TITLE
Clarify "minimal jitter" in importing_scenes tutorial

### PR DESCRIPTION
**Pull request by clayjohn**
This addresses issue #1913

### CODE REVIEW

File: getting_started/workflow/assets/importing_scenes.rst [hunk 1]
```diff
@@ -224,7 +224,7 @@ FPS
 
 Most 3D export formats store animation timeline in seconds instead of frames. To ensure
 animations are imported as faithfully as possible, please specify the frames per second
-used to edit them. Failing to do this may result in minimal jitter.
+used to edit them. Failing to do this may result in a small amount of jitter.
```
Comment (akien-mga) [inline getting_started/workflow/assets/importing_scenes.rst]: "Jitter" isn't really countable so it's strange to me to speak about "an amount of jitter". I'd remove the qualifier altogether.

Comment (clayjohn) [reply getting_started/workflow/assets/importing_scenes.rst]: What about "perceptible jittering"?

Comment (clayjohn) [reply getting_started/workflow/assets/importing_scenes.rst]: Actually what about "may result in shaky animations"? I have a feeling that jitter isn't going to translate well, and it's a kinda awkward word in English anyway.

Comment (akien-mga) [reply getting_started/workflow/assets/importing_scenes.rst]: Sounds good.

**Comment by clayjohn** [comment]
@akien-mga Should be good to go now!

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**